evo_researcher/benchmark/utils.py

```diff
@@ -53,19 +61,23 @@ def load(cls, markets: t.List[Market], path: str):
         }
 
 
-def get_manifold_markets(number: int = 100) -> t.List[Market]:
+def get_manifold_markets(
+    number: int = 100, excluded_questions: t.List[str] = []
+) -> t.List[Market]:
     url = "https://api.manifold.markets/v0/search-markets"
     params = {
         "term": "",
         "sort": "liquidity",
         "filter": "open",
-        "limit": f"{number}",
+        "limit": f"{number + len(excluded_questions)}",
```

Comment:
Won't this fail if the number of excluded questions gets too large? It's not in Polywrap's implementation either, so I think it can be removed.

Reply:
Not sure I understand, sorry? Just had a look, and it seems like the max number of markets you can get at once is 1000, so do you mean add a check that `number + len(excluded_questions) < 1000`?

Reply:
1000 sounds like a good enough reserve 😄  

But the second thing I meant is that for manifold, we have `limit": f"{number + len(excluded_questions)}",` but for polymarket, there is `?_limit={number}` and there is also this check

```
if number > 100:
        raise ValueError("Polymarket API only returns 100 markets at a time")
```

so there isn't the reserve to ask for extra markets. The behaviour between the two functions is different, just thought that it would be nice to have it consistent, as these will be used at the same places.

Reply:
Ah sorry, yeah the polymarket api call was missing the extra `len(excluded_questions)`. Have fixed.